app/css/ui-kit/buttons.css

```diff
@@ -231,4 +231,10 @@
             @apply text-15;
         }
     }
+
+    .btn-casual {
+        @apply text-textColor1 font-semibold text-18;
+        @apply border-1 border-textColor1 bg-white rounded-8;
+        @apply shadow-xsZ1v3;
+    }
```

Comment:
We shouldn't include font-sizes, rounding, etc in button classes like this. The text-18 and rounded-8 etc should come from adding the `btn-l` to the element. I also think in this case, the actual button is `btn-default` or `btn-enhanced`. I think the only difference is the shadow?

Reply:
There are a couple differences between any of those and this one in the design. 

> We shouldn't include font-sizes, rounding, etc in button classes like this. The text-18 and rounded-8 etc should come from adding the btn-l to the element. 

Ok!